README.rst

```diff
@@ -33,14 +35,14 @@ Project Status
     :target: https://travis-ci.org/astropy/astropy
     :alt: Astropy's Travis CI Status
 
-.. image:: https://coveralls.io/repos/astropy/astropy/badge.svg
-    :target: https://coveralls.io/r/astropy/astropy
-    :alt: Astropy's Coveralls Status
-
 .. image:: https://ci.appveyor.com/api/projects/status/ym7lxajcs5qwm31e/branch/master?svg=true
     :target: https://ci.appveyor.com/project/Astropy/astropy/branch/master
     :alt: Astropy's Appveyor Status
 
+.. image:: https://coveralls.io/repos/astropy/astropy/badge.svg
+    :target: https://coveralls.io/r/astropy/astropy
+    :alt: Astropy's Coveralls Status
+
 For an overview of the testing and build status of all packages associated
 with the Astropy Project, see http://dashboard.astropy.org.
 
```

Comment:
I had to read the readme 4 times before I saw the badge. I think it should go down in the Project Status section, next to "powered by Numfocus" and some text should be added. Here's some text inspired by the text on the NumFocus donate page:

    The Astropy Project is sponsored by NumFOCUS, a 501(c)(3) nonprofit in the United States. Your tax-deductible donation to the Astropy Project will support our mission to promote sustainable high-level code base for the astronomy community, open code development, educational materials, and reproducible scientific research.